lib/Net/Stripe.pm

```diff
@@ -75,11 +75,12 @@ You can set this to true to see the actual network requests.
 
 =cut
 
-has 'debug'         => (is => 'rw', isa => 'Bool',   default    => 0, documentation => "The debug flag");
-has 'debug_network' => (is => 'rw', isa => 'Bool',   default    => 0, documentation => "The debug network request flag");
-has 'api_key'       => (is => 'ro', isa => 'Str',    required   => 1, documentation => "You get this from your Stripe Account settings");
-has 'api_base'      => (is => 'ro', isa => 'Str',    lazy_build => 1, documentation => "This is the base part of the URL for every request made");
-has 'ua'            => (is => 'ro', isa => 'Object', lazy_build => 1, documentation => "The LWP::UserAgent that is used for requests");
+has 'debug'         => (is => 'rw', isa => 'Bool',   default    => 0,  documentation => "The debug flag");
+has 'debug_network' => (is => 'rw', isa => 'Bool',   default    => 0,  documentation => "The debug network request flag");
+has 'api_key'       => (is => 'ro', isa => 'Str',    required   => 1,  documentation => "You get this from your Stripe Account settings");
+has 'api_base'      => (is => 'ro', isa => 'Str',    lazy_build => 1,  documentation => "This is the base part of the URL for every request made");
+has 'api_version'   => (is => 'ro', isa => 'Str',    default    => '', documentation => "This is the value of the Stripe-Version header you wish to use for API calls");
```

Comment:
Am I right in thinking this line is the only "functional" change between lines 78 and 83? If that's the case, could you please create a new pull request with only the changes affecting the code's behaviour? I apologise for sounding finicky but if things go wrong I want it to be easy to see exactly what has changed in each commit. Thanks!

Reply:
@andrewsolomon, yes, the only material change was line 82. i was attempting to follow what i saw as the alignment/indentation protocols, based on the previous commits i have browsed through. i have no problem with "finicky" because i too hate sifting through whitespace-only diffs in order to identify the material changes.

so, can you provide some guidance as to your personal preference in this type of situation? ie, line 82 ends up being one column wider than the others because of the two-character default value. so would you:
- insert 82 as-is and where-is, leaving the misalignment?

- add 82 to the bottom of the list, leaving the misalignment?

- other?

also, being _very_ new to git, it is not obvious to me how to produce the requested new pull request once i have received your feedback. do i create a new branch starting with the upstream master and make my edits there, do i checkout my existing branch, edit it, and re-commit, etc? and please, type very very slowly so i can understand. :-)

thanks

Reply:
I'm happy with either of the the first two options. You choose :-)

Regarding git and public open source code, I prefer each commit in the master branch to be a complete change we're happy to release. For this reason the approach I'd take is to:
* commit/hack/commit away on this branch until you're happy
* squash the commits into a single commit  - rewriting history
* push your local branch to a new branch your github repo and do a pull request on that branch.

Sounds convoluted, but it's not too bad. Let me know when you're happy with the code on this branch and I can walk you through it.

Reply:
i think i should be able to manage that. i will definitely chime back in if i  need specific instruction.

thanks